files/usr/local/bin/mgr/utils.lua

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
for the mgr.utils, i think some of these are duplicated in /usr/lib/lua/aredn/utils.lua.  why not just require("aredn.utils")?

Reply:
mgr.utils is a bit of a dumping ground at the moment (and yes, some duplicates or near duplicates in there too). I don't expect it to survive to the end.